atomic_reactor/plugins/exit_sendmail.py

```diff
@@ -269,6 +275,9 @@ def _get_receivers_list(self):
         # Remove duplicates
         receivers_list = list(set(receivers_list))
 
+        # Remove empty and None items
+        receivers_list = [x for x in receivers_list if x]
```

Comment:
I 'd rather have this in two separate lines - combining those would produce a hard-to-read mess